github/Requester.py

```diff
@@ -89,13 +89,18 @@ def read(self):
 
 class HTTPSRequestsConnectionClass(object):
     # mimic the httplib connection object
-    def __init__(self, host, port=None, strict=False, timeout=None, **kwargs):
+    def __init__(self, host, port=None, strict=False, timeout=None, retry=None, **kwargs):
         self.port = port if port else 443
         self.host = host
         self.protocol = "https"
         self.timeout = timeout
         self.verify = kwargs.get("verify", True)
         self.session = requests.Session()
+        # Code to support retries
+        if retry != None:
```

Comment:
Instead of __if retry != None:__, it might be safer to use __if retry:__ to deal with the case where the if the caller passes in __retry=False__.

Reply:
We check if the `retry` parameter is an `int' at https://github.com/PyGithub/PyGithub/pull/1002/files#diff-408d795214c69592c37ec1f499122deaR122

But I guess it wouldn't hurt to change this to `if retry:` since `retry=0` would be the same as no retry.